packages/main/src/Select.ts

```diff
@@ -385,7 +392,7 @@ class Select extends UI5Element {
 		this.responsivePopover = await this._respPopover();
 
 		this.options.forEach(option => {
-			option._getRealDomRef = () => this.responsivePopover.querySelector(`*[data-ui5-stable=${option.stableDomRef}]`);
+			option._getRealDomRef = () => this.responsivePopover.querySelector(`*[data-ui5-stable=${option.stableDomRef}]`) as HTMLElement;
```

Comment:
The `querySelector<HTMLElement>` is preferred over the cast.

Note: when you change  it to the generic, you may also need to add "!" at the end of the expression (or not , depending on your use-case).
Because "as HTMLElement" is doing two things, setting the type, but also setting the value as not null, undefined (as if "!" is automatically added).